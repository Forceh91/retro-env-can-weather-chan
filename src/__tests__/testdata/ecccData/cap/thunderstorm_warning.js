export default `<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<alert xmlns="urn:oasis:names:tc:emergency:cap:1.2">
    <identifier>urn:oid:2.49.0.1.124.1211373515.2023</identifier>
    <sender>cap-pac@canada.ca</sender>
    <sent>2023-08-05T22:44:46-00:00</sent>
    <status>Actual</status>
    <msgType>Update</msgType>
    <source>Env. Can. - Can. Met. Ctr. – Montréal</source>
    <scope>Public</scope>
    <code>profile:CAP-CP:0.4</code>
    <code>layer:SOREM:1.0</code>
    <code>layer:EC-MSC-SMC:1.0</code>
    <code>layer:EC-MSC-SMC:1.1</code>
    <code>layer:SOREM:2.0</code>
    <note>Service Notice - February 2023: The Environment and Climate Change Canada (ECCC) CAP Service undergoes changes from time to time as the business of alerting evolves. For 2023, changes will include... 1) modifications, or even removal, of the Wireless Public Alerting Message (WPAM) layer due to the successful pilot of the UpdateX feature recently deployed in the Canadian Cell Broadcasting Environment, and 2) other minor improvements and corrections. For more information on these changes, you are invited to subscribe to the following mailing list: http://lists.cmc.ec.gc.ca/mailman/listinfo/dd_info | Notification de service - février 2023: Le service du PAC d’Environnement et Changement climatique Canada (ECCC) subit périodiquement des changements et ajouts à mesure que le système d’alerte évolue. Pour 2023, ces changements incluent... 1) des modifications, voire la suppression, de la couche du Service d’alertes sans fil au public (WPAM) en raison du succès du projet pilote de la fonctionnalité UpdateX récemment déployée dans l’environnement de diffusion sans fil canadien, et 2) d’autres améliorations et corrections mineures. Pour plus d’informations sur ces changements, vous êtes invités à vous inscrire à la liste de diffusion suivante: http://lists.cmc.ec.gc.ca/mailman/listinfo/dd_info</note>
    <references>cap-pac@canada.ca,urn:oid:2.49.0.1.124.3723074918.2023,2023-08-05T20:10:43-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.1149892027.2023,2023-08-05T20:14:33-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2316707614.2023,2023-08-05T20:19:04-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2348193114.2023,2023-08-05T20:34:27-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2845094716.2023,2023-08-05T20:39:26-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.3543408946.2023,2023-08-05T21:45:12-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.1228018707.2023,2023-08-05T21:50:13-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.0657098959.2023,2023-08-05T22:37:22-00:00</references>
    <info>
        <language>en-CA</language>
        <category>Met</category>
        <event>thunderstorm</event>
        <responseType>Monitor</responseType>
        <urgency>Immediate</urgency>
        <severity>Moderate</severity>
        <certainty>Likely</certainty>
        <audience>general public</audience>
        <eventCode>
            <valueName>profile:CAP-CP:Event:0.4</valueName>
            <value>thunderstorm</value>
        </eventCode>
        <eventCode>
            <valueName>SAME</valueName>
            <value>SVR</value>
        </eventCode>
        <effective>2023-08-05T22:37:22-00:00</effective>
        <expires>2023-08-06T01:36:22-00:00</expires>
        <senderName>Environment Canada</senderName>
        <headline>severe thunderstorm warning in effect</headline>
        <description>
At 5:37 p.m. CDT (6:37 p.m. EDT), Environment Canada meteorologists are tracking a severe thunderstorm capable of producing strong wind gusts and up to nickel size hail.

This severe thunderstorm is located near Flindt Lake, moving southeast at 30 km/h. 

Hazard: Strong wind gusts and nickel size hail.

Locations impacted include:
Flet Lake, Flindt Lake and Allan Water.

Another thunderstorm is located 5 kilometres southwest of Vincent Lake, moving east at 45 km/h. 

Hazard: Strong wind gusts and nickel size hail.

Locations impacted include:
St. Raphael Lake, Hooker Lake, Vincent Lake and Hill Lake.

###

Large hail can damage property and cause injury. Strong wind gusts can toss loose objects, damage weak buildings, break branches off trees and overturn large vehicles. Locally heavy rain is also possible.

Severe thunderstorm warnings are issued when imminent or occurring thunderstorms are likely to produce or are producing one or more of the following: large hail, damaging winds, torrential rainfall.
</description>
        <instruction>
Lightning kills and injures Canadians every year. Remember, when thunder roars, go indoors!

Emergency Management Ontario recommends that you take cover immediately if threatening weather approaches.
</instruction>
        <web>http://weather.gc.ca/warnings/index_e.html</web>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Type</valueName>
            <value>warning</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Broadcast_Intrusive</valueName>
            <value>no</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:1.0:Broadcast_Immediately</valueName>
            <value>No</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Parent_URI</valueName>
            <value>msc/alert/environment/hazard/alert-3.0-ascii/consolidated-xml-2.0/20230805223722118/ON_00_01_LAND/STW/201043373865160756202308050503_ON_00_01_LAND/actual/en_proper_complete_u-fr_proper_complete_c/NinJo</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:CAP_count</valueName>
            <value>A:7846 M:43254 C:52366</value>
        </parameter>
        <parameter>
            <valueName>profile:CAP-CP:0.4:MinorChange</valueName>
            <value>text</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Location_Status</valueName>
            <value>active</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Name</valueName>
            <value>severe thunderstorm warning</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Coverage</valueName>
            <value>Ontario</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:1.0:Broadcast_Text</valueName>
            <value>At 6:37 p.m. Eastern Daylight Time Saturday, Environment Canada has issued a severe thunderstorm warning for Northwestern and North of Superior Ontario. Extra care and attention should be exercised in and around the following localities: Savant Lake, Sturgeon Lake, Armstrong, Auden and Wabakimi Park. Environment Canada meteorologists are tracking a severe thunderstorm capable of producing strong wind gusts and up to nickel size hail. Take cover immediately if threatening weather approaches. Intense lightning is likely and tornados are possible with any thunderstorm that develops.</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Designation_Code</valueName>
            <value>ON_00_01_LAND</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:2.0:WirelessImmediate</valueName>
            <value>No</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:2.0:WirelessText</valueName>
            <value>At 6:37 p.m. Eastern Daylight Time Saturday, Environment Canada has issued a severe thunderstorm warning for this mobile coverage area. Take cover immediately if threatening weather approaches.</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Alert_Location_Status</valueName>
            <value>active</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Newly_Active_Areas</valueName>
            <value>048210</value>
        </parameter>
        <area>
            <areaDesc>Savant Lake - Sturgeon Lake</areaDesc>
            <polygon>49.9131,-90.0123 49.5235,-90.0178 49.5614,-90.2392 49.8681,-91.6834 49.8683,-91.6842 51.1282,-91.0434 51.1354,-89.9953 49.9131,-90.0123</polygon>
            <geocode>
                <valueName>layer:EC-MSC-SMC:1.0:CLC</valueName>
                <value>047210</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558080</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558085</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558090</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3560090</value>
            </geocode>
        </area>
        <area>
            <areaDesc>Armstrong - Auden - Wabakimi Park</areaDesc>
            <polygon>49.9338,-88.5445 49.9131,-90.0123 51.1354,-89.9953 51.1568,-87.9963 51.159,-87.4321 49.9482,-87.4321 49.9338,-88.5445</polygon>
            <geocode>
                <valueName>layer:EC-MSC-SMC:1.0:CLC</valueName>
                <value>048210</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558090</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558097</value>
            </geocode>
        </area>
    </info>
    <info>
        <language>fr-CA</language>
        <category>Met</category>
        <event>orages</event>
        <responseType>Monitor</responseType>
        <urgency>Immediate</urgency>
        <severity>Moderate</severity>
        <certainty>Likely</certainty>
        <audience>grand public</audience>
        <eventCode>
            <valueName>profile:CAP-CP:Event:0.4</valueName>
            <value>thunderstorm</value>
        </eventCode>
        <eventCode>
            <valueName>SAME</valueName>
            <value>SVR</value>
        </eventCode>
        <effective>2023-08-05T22:37:22-00:00</effective>
        <expires>2023-08-06T01:36:22-00:00</expires>
        <senderName>Environnement Canada</senderName>
        <headline>alerte d'orages violents en vigueur</headline>
        <description>
À 17h37 HAC (18h37 HAE), les météorologues d’Environnement Canada surveillent un orage violent pouvant produire des rafales fortes et de la grêle pouvant atteindre la taille d’une pièce de cinq cents.

Cet orage violent est situé près du lac Flindt et se dirige vers le sud-est à 30 km/h.

Dangers : fortes rafales et grêle de la taille d'un 5 cents.

Les régions touchées comprennent :
lac Flet, lac Flindt et Allan Water.

Un autre orage se trouve à 5 kilomètres au sud-ouest du lac Vincent et se dirige vers l'est à 45 km/h.

Dangers : fortes rafales et grêle de la taille d'un 5 cents.

Les régions touchées comprennent :
lac St. Raphael, lac Hooker, lac Vincent et lac Hill.

###

De la grosse grêle peut endommager des biens et causer des blessures. De fortes rafales peuvent projeter des objets non fixés, endommager des bâtiments peu solides, arracher des branches d'arbres et renverser de gros véhicules. De la pluie forte par endroits est également possible.

Une alerte d'orages violents est émise lorsqu'il y a ou qu'il y aura des orages qui s'accompagnent ou qui s'accompagneront probablement d'au moins un des éléments suivants : grosse grêle, vents destructeurs ou pluies torrentielles.
</description>
        <instruction>
La foudre tue et blesse des Canadiens chaque année. N'oubliez pas : quand le tonnerre gronde, rentrez vite à l'intérieur!

Gestion des situations d'urgence Ontario vous recommande de trouver un abri immédiatement si un phénomène météorologique menaçant est imminent.
</instruction>
        <web>http://meteo.gc.ca/warnings/index_f.html</web>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Type</valueName>
            <value>warning</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Broadcast_Intrusive</valueName>
            <value>no</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:1.0:Broadcast_Immediately</valueName>
            <value>No</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Parent_URI</valueName>
            <value>msc/alert/environment/hazard/alert-3.0-ascii/consolidated-xml-2.0/20230805223722118/ON_00_01_LAND/STW/201043373865160756202308050503_ON_00_01_LAND/actual/en_proper_complete_u-fr_proper_complete_c/NinJo</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:CAP_count</valueName>
            <value>A:7846 M:43254 C:52366</value>
        </parameter>
        <parameter>
            <valueName>profile:CAP-CP:0.4:MinorChange</valueName>
            <value>text</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Location_Status</valueName>
            <value>active</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Name</valueName>
            <value>alerte d'orages violents</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.0:Alert_Coverage</valueName>
            <value>Ontario</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:1.0:Broadcast_Text</valueName>
            <value>À 18h37 heure avancée de l'Est le samedi, Environnement Canada a émis une alerte d'orages violents pour le nord-ouest et le nord du Supérieur en l'Ontario. Une attention et des précautions additionnelles devraient être exercées près et autour des localités suivantes: Savant Lake, le lac Sturgeon, Armstrong, Auden et le parc Wabakimi. Les météorologues d’Environnement Canada surveillent un orage violent pouvant produire des rafales fortes et de la grêle pouvant atteindre la taille d’une pièce de cinq cents. Mettez-vous immédiatement à l'abri si un phénomène météorologique menaçant s'approche. De nombreux éclairs sont probables et il y a toujours un risque de tornade avec un orage.</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Designation_Code</valueName>
            <value>ON_00_01_LAND</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:2.0:WirelessImmediate</valueName>
            <value>No</value>
        </parameter>
        <parameter>
            <valueName>layer:SOREM:2.0:WirelessText</valueName>
            <value>À 18h37 heure avancée de l'Est le samedi, Environnement Canada a émis une alerte d'orages violents pour cette zone de couverture mobile. Mettez-vous immédiatement à l'abri si un phénomène météorologique menaçant s'approche.</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Alert_Location_Status</valueName>
            <value>active</value>
        </parameter>
        <parameter>
            <valueName>layer:EC-MSC-SMC:1.1:Newly_Active_Areas</valueName>
            <value>048210</value>
        </parameter>
        <area>
            <areaDesc>Savant Lake - lac Sturgeon</areaDesc>
            <polygon>49.9131,-90.0123 49.5235,-90.0178 49.5614,-90.2392 49.8681,-91.6834 49.8683,-91.6842 51.1282,-91.0434 51.1354,-89.9953 49.9131,-90.0123</polygon>
            <geocode>
                <valueName>layer:EC-MSC-SMC:1.0:CLC</valueName>
                <value>047210</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558080</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558085</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558090</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3560090</value>
            </geocode>
        </area>
        <area>
            <areaDesc>Armstrong - Auden - parc Wabakimi</areaDesc>
            <polygon>49.9338,-88.5445 49.9131,-90.0123 51.1354,-89.9953 51.1568,-87.9963 51.159,-87.4321 49.9482,-87.4321 49.9338,-88.5445</polygon>
            <geocode>
                <valueName>layer:EC-MSC-SMC:1.0:CLC</valueName>
                <value>048210</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558090</value>
            </geocode>
            <geocode>
                <valueName>profile:CAP-CP:Location:0.3</valueName>
                <value>3558097</value>
            </geocode>
        </area>
    </info>
<Signature xmlns="http://www.w3.org/2000/09/xmldsig#" Id="Environment Canada">
<SignedInfo>
<CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/>
<SignatureMethod Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"/>
<Reference URI="">
<Transforms>
<Transform Algorithm="http://www.w3.org/2000/09/xmldsig#enveloped-signature"/>
</Transforms>
<DigestMethod Algorithm="http://www.w3.org/2001/04/xmlenc#sha256"/>
<DigestValue>l4ZC0KrBbTdDyjJ8MhltDzCwC71tG440TPXyyam6NgU=</DigestValue>
</Reference>
</SignedInfo>
<SignatureValue>vuQNBhGsKeArOBiTjY7x0RhGxqNCeT1u6SS3rbWFrDuNUNz6ZrcX6noVK1O5p2Vz
i+2xR/JlqFxCjTm2FUpgazA9yx7TJeD2+4/Xl9/ZBFO/uG8gM3fnGzSzPy03bj5K
oK4upZ1sJbM6dL0V9WBoSXj/szc4qkqWSg6/5nhk6HmPp/EF1qOCuppqSYlH8oPP
UoyaUn325lESXJ7Pk7KLk9UFZDnFZy7ZsuRyTmspWedMQgSPdVRkbw/OjaS57IMT
gkN71zndRA+52biRx20iqN0G4ulH+DxVfKW1m/BMA1BWtq3QTtS5LKwgZ2k+Gzdd
7MVgPDLeFBIflrEPXG1FHg==</SignatureValue>
<KeyInfo>
<X509Data>
<X509Certificate>MIIHQDCCBiigAwIBAgIQauIRkpOPcGViI+LoZHWDFzANBgkqhkiG9w0BAQsFADCB
ujELMAkGA1UEBhMCVVMxFjAUBgNVBAoTDUVudHJ1c3QsIEluYy4xKDAmBgNVBAsT
H1NlZSB3d3cuZW50cnVzdC5uZXQvbGVnYWwtdGVybXMxOTA3BgNVBAsTMChjKSAy
MDEyIEVudHJ1c3QsIEluYy4gLSBmb3IgYXV0aG9yaXplZCB1c2Ugb25seTEuMCwG
A1UEAxMlRW50cnVzdCBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eSAtIEwxSzAeFw0y
MzA0MjcxNDEyNDVaFw0yNDA1MDcxNDEyNDVaMGwxCzAJBgNVBAYTAkNBMQ8wDQYD
VQQIEwZRdWViZWMxETAPBgNVBAcTCEdhdGluZWF1MR8wHQYDVQQKExZTaGFyZWQg
U2VydmljZXMgQ2FuYWRhMRgwFgYDVQQDDA8qLndlYXRoZXIuZ2MuY2EwggEiMA0G
CSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDOSYFvWlJgQgD5twPwuMqUXQB8Raj6
enUh60KlOqYi2Moja8qBMhANS848XjHd+OlTNKv5J9YT6OoxTqcRe60e3nWYF6/u
Plu+O4KWOO8x6jWbttEn5G7j7V35yaWXqP0LObVrQLDMYX3D4xsm+MJK7eyGJz8k
AJAJYONS/+dp7/0jlrzlTjyuT4Uk50zVDvTGxiYpgjPfuXG9Bke5q8Cju4EEgAfW
1I2+67Is7ae2fTjhvPH+w+L4iujk1ucmkrZqd2Norr8eeLebvDiSNxV4FkdzUZKQ
TTycHQb1u/79WGgRKCH+nPYTzW/l/muRz59V0LKTQOlFmq8H45fmSYWZAgMBAAGj
ggONMIIDiTAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBRlq9u/A7i58xA3Eb4DmYis
FzYwKTAfBgNVHSMEGDAWgBSConB03bxTP8971PfNf6dgxgpMvzBoBggrBgEFBQcB
AQRcMFowIwYIKwYBBQUHMAGGF2h0dHA6Ly9vY3NwLmVudHJ1c3QubmV0MDMGCCsG
AQUFBzAChidodHRwOi8vYWlhLmVudHJ1c3QubmV0L2wxay1jaGFpbjI1Ni5jZXIw
MwYDVR0fBCwwKjAooCagJIYiaHR0cDovL2NybC5lbnRydXN0Lm5ldC9sZXZlbDFr
LmNybDCBnQYDVR0RBIGVMIGSgg8qLndlYXRoZXIuZ2MuY2GCDXdlYXRoZXIuZ2Mu
Y2GCC21ldGVvLmdjLmNhgg0qLm1ldGVvLmdjLmNhghUqLmFscGhhLndlYXRoZXIu
Z2MuY2GCEyouYWxwaGEubWV0ZW8uZ2MuY2GCFCouYmV0YS53ZWF0aGVyLmdjLmNh
ghIqLmJldGEubWV0ZW8uZ2MuY2EwDgYDVR0PAQH/BAQDAgWgMB0GA1UdJQQWMBQG
CCsGAQUFBwMBBggrBgEFBQcDAjBMBgNVHSAERTBDMDcGCmCGSAGG+mwKAQUwKTAn
BggrBgEFBQcCARYbaHR0cHM6Ly93d3cuZW50cnVzdC5uZXQvcnBhMAgGBmeBDAEC
AjCCAXsGCisGAQQB1nkCBAIEggFrBIIBZwFlAHUAc9meiRtMlnigIH1HneayxhzQ
UV5xGSqMa4AQesF3crUAAAGHww80IQAABAMARjBEAiAkpowR7bbTYLp4v/tw0Fav
HHP8mm2i6/ecg03yasJkkgIga1hmg5g/Gq97+HOCDwjsy77aS8+PirSarIO6/X1H
EfkAdQA7U3d1Pi25gE6LMFsG/kA7Z9hPw/THvQANLXJv4frUFwAAAYfDDzQcAAAE
AwBGMEQCIBKVCiUTam6W9Y+5TiuhpsxVVx2TYVz9SebYPuB4XyHUAiB4Y9ZXnkOt
rJJAaw4im3kY7OWZG40xPUL1ygK8trUsIAB1AO7N0GTV2xrOxVy3nbTNE6Iyh0Z8
vOzew1FIWUZxH7WbAAABh8MPNF0AAAQDAEYwRAIgYbZkNJlz8a4d35p4+wLg0h5P
WtFyyBEqy06IoSV7UWICIH65mirJoLVsZFK7kXW3IJF4+xlgATBcr/n2ImBLSWug
MA0GCSqGSIb3DQEBCwUAA4IBAQCcp2dfEDsCzJHpMsTF1hlEyYPTgmfrSbgegWT3
76r0r7XGY7qIym53HE6Pk3Q7BgsegpV+pS8UMWx1m9x0ZcUQX11gMBlNLv0WCSBe
4/9BOaGc2deYrXsRAmmFJwqvgegcBSQhg4zZSoSSVqJPhaJ1WYWqDbJW+doY15xy
JPm7UbZqCAbzm3/c36N9j/Zu6h+LA9rhltfwYoWCyqlBtJCRrs7K7KEP+dLtsoby
+AhyY+3dK2IB4G7v5Qbxa88WKV9EsXXMpvl0v/kXSDjIHJwRPyOf1OKU9x2Mc3Mx
KEgzSJ0GUUW0RqqCVWZ14TBoxUq1jJw4m12zSF+PYFpSH6g5</X509Certificate>
</X509Data>
</KeyInfo>
</Signature></alert>
`;
