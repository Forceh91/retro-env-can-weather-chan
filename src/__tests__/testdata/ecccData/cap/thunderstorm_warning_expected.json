{
  "identifier": "urn:oid:2.49.0.1.124.1211373515.2023",
  "sender": "cap-pac@canada.ca",
  "references": "cap-pac@canada.ca,urn:oid:2.49.0.1.124.3723074918.2023,2023-08-05T20:10:43-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.1149892027.2023,2023-08-05T20:14:33-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2316707614.2023,2023-08-05T20:19:04-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2348193114.2023,2023-08-05T20:34:27-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.2845094716.2023,2023-08-05T20:39:26-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.3543408946.2023,2023-08-05T21:45:12-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.1228018707.2023,2023-08-05T21:50:13-00:00 cap-pac@canada.ca,urn:oid:2.49.0.1.124.0657098959.2023,2023-08-05T22:37:22-00:00",
  "effective": "2023-08-05T22:37:22-00:00",
  "expires": "2023-08-06T01:36:22-00:00",
  "headline": "severe thunderstorm warning in effect",
  "description": "At 5:37 p.m. CDT (6:37 p.m. EDT), Environment Canada meteorologists are tracking a severe thunderstorm capable of producing strong wind gusts and up to nickel size hail.",
  "instruction": "Lightning kills and injures Canadians every year. Remember, when thunder roars, go indoors!",
  "severity": 2,
  "urgency": 4,
  "event": "thunderstorm",
  "certainty": "Likely",
  "audience": "general public",
  "areas": [
    [49.9131, -90.0123],
    [49.5235, -90.0178],
    [49.5614, -90.2392],
    [49.8681, -91.6834],
    [49.8683, -91.6842],
    [51.1282, -91.0434],
    [51.1354, -89.9953],
    [49.9131, -90.0123]
  ]
}
